projects/js-toolkit/packages/js-toolkit-scripts/src/scripts/create-jar/xml.ts

```diff
@@ -22,27 +22,27 @@ const TYPES = {
 export function addMetatypeAttr(
 	metatype: XmlObject,
 	id: string,
-	desc: ConfigurationJsonField
+	description
```

Comment:
Looks like we accidentally removed the type declaration for this re-write. This shouldn't be a future issue for other people if we don't add autofix, but we need to fix this here.

Reply:
Interesting case: because the fix doesn't take into account TypeScript it fails to maintain the type declaration :scream: . This is one of the more difficult issues of dealing with ASTs: that you never know in advance how many constructs there may be, unless you read the full spec from line 1 to last (and even in that case you could forget one unwittingly :-( ).